src/main/java/com/ldtteam/structurize/helpers/Settings.java

```diff
@@ -348,8 +353,7 @@ public void setActiveSchematic(final Blueprint blueprint)
     public void reset()
     {
         resetBlueprint();
-        rotation = 0;
-        isMirrored = false;
+        placement = new PlacementSettings();
```

Comment:
Personally, I'd put the Wall stuff into a "WallSettings" which you can get from here, makes this much less cluttered.

Reply:
That is already the case.  `PlacementSettings` holds a `WallExtents`.